server/models/Collective.js

```diff
@@ -1621,8 +1621,17 @@ export default function(Sequelize, DataTypes) {
           return models.Member.update({ deletedAt: new Date() }, { where: { id: { [Op.in]: diffMemberIds } } });
         }
       })
-      .then(() => {
-        return Promise.map(members, member => {
+      .then(async () => {
+        const oldAdminMembers = await this.getMembers({
+          where: { role: { [Op.in]: [roles.ADMIN] } },
+        });
+        const adminMembers = members.filter(m => m.role === roles.ADMIN);
+        const diff = differenceBy(oldAdminMembers, adminMembers, 'role');
+
+        if (oldAdminMembers.length === 1 && diff.length !== 0) {
```

Comment:
I don't really understand this, will it work if there are two admins in my collective and I change the role to `MEMBER` for both of them?